views/pages/fixture_editor.js

```diff
@@ -30,81 +30,69 @@ module.exports = function(options) {
 
   // Existing manufacturer
   str += `<section v-if="fixture.useExistingManufacturer">`;
-  str += `<label class="validate-group">`;
-  str += `<span class="label">Choose from list</span>`;
-  str += `<span class="value">`;
-  str += `<select required v-model="fixture.manufacturerShortName" :class="{ empty: fixture.manufacturerShortName === '' }" ref="existingManufacturerSelect">`;
-  str += `<option value="">Please select a manufacturer</option>`;
-  for (const man of Object.keys(options.register.manufacturers)) {
-    str += `<option value="${man}">${options.manufacturers[man].name}</option>`;
-  }
-  str += `</select>`;
-  str += `<span class="error-message" hidden></span>`;
-  str += `</span>`;
-  str += `</label>`;
+  str += simpleLabel(
+    `Choose from list`,
+    [
```

Comment:
Better construct the string into a local variable first and then just call the function with it.